Header component

GitJustinLau opened the pull request.

GitJustinLau committed:
create header component
create searchbar component

GitJustinLau committed:
create button component
add types to header component

koo-mi commented:
Great job implementing tailwind and defining types. 
However, I think Header still requires more styles to resemble the mockup.

GitJustinLau committed:
remove scss page (will be using tailwind)

GitJustinLau committed:
move header and SearchBar out of folders.
put header component into layout
remove header from page page
additional styling for header and SearchBar

GitJustinLau committed:
Merge branch 'develop' into header-component

GitJustinLau committed:
fix svg urls

koo-mi commented:
Great job styling the header! 

I think it's good to merge.
